terms_test.go

```diff
@@ -39,6 +39,9 @@ func TestTermsString(t *testing.T) {
 	}{
 		{`off`, []string{""}},
 		{`2*return("abc")`, []string{"abc", "abc", ""}},
+		{`100%return("abc")`, []string{"abc", "abc", "abc"}},
+		{`100.0%return("abc")`, []string{"abc", "abc", "abc"}},
+		{`100%2*return("abc")`, []string{"abc", "abc", ""}},
```

Comment:
Could you add a test for `0%`?

Reply:
added in https://github.com/pingcap/failpoint/pull/40/commits/ab64c073b93cb0f9e212d5e7478a01ae4d5a5710